session-1/oscar-arrieta/buscar_arreglo.rb

```diff
@@ -0,0 +1,15 @@
+def buscar(a = [], string)
+  b = string
+  indice = a.index(b)
+ if indice
+  "#{ indice }"
+ else
+  "#{ -1 }"
+ end
+end
```

Comment:
avoid to use no readable names for variables for example **array** instead **a**, maybe this ternary operator works better, wdyt?
```suggestion
def buscar(array = [], element)
  array.include?(element) ? array.index(element) : -1
end
```

Reply:
done